Export Angular provider

## Prerequisites

* Version: `0.176.0`
* Are you running from source/main: no
* Are you using a released build: yes

## Step to reproduce

Try to import `LucideIconProvider` and `LUCIDE_ICONS` injection token from `lucide-angular`.

### Actual behavior

Error: `Module not found: Error: Package path ./lib/lucide-icon.provider is not exported from package`

## Notes

I have a custom directive which creates a `lucide-angular` icon.  I'd like to access the injected icons from the new provider.  To do that, I need the provider to be exported.

--- a/packages/lucide-angular/src/public-api.ts
+++ b/packages/lucide-angular/src/public-api.ts
@@ -3,5 +3,6 @@ import * as icons from './icons/lucide-icons';
 export * from './lib/lucide-angular.component';
 export * from './lib/lucide-angular.module';
 export * from './lib/lucide-icon.config';
+export * from './lib/lucide-icon.provider';
 export * from './icons/lucide-icons';
 export { icons };

